db-test/tests/db/db/query.rkt

```diff
@@ -564,7 +565,53 @@
       (with-connection c
         (check-exn exn:fail? (lambda () (query-value c "select nonsuch from notthere")))
         (check-equal? (query-value c (select-val "17"))
-                      (if (ANDFLAGS 'odbc 'issl) "17" 17))))))
+                      (if (ANDFLAGS 'odbc 'issl) "17" 17))))
+
+    ;; Added 23 May 2017.
+    ;; In certain cases malformed queries do not cause expected syntax errors to be raised
+    ;; instead they trigger a 'multiple statements given' error which is misleading.
+    ;; Fixed so correct errors are raised.
+    (unless (or (ANDFLAGS 'odbc 'ispg) (ORFLAGS 'isdb2))
+      (test-case "malformed query do not cause multiple statements in string error"
+        (with-connection c
+          ; Verify multiple statements given error is not the exn:fail that was raised.
+          (check-exn exn:fail-is-not-multiple-statements-error?
+                 (lambda () (query-exec c "Select ,* from nonexistenttable")))
```

Comment:
The `(lambda` should be indented to the same column as `exn:fail`  (DrRacket-style)